core/string/translation.cpp

```diff
@@ -35,6 +35,11 @@
 #include "core/os/os.h"
 
 #ifdef TOOLS_ENABLED
+#include "core/io/compression.h"
+#include "core/io/file_access_memory.h"
+#include "core/io/translation_loader_po.h"
+#include "editor/doc_translations.gen.h"
+#include "editor/editor_translations.gen.h"
```

Comment:
`core` shouldn't depend on `editor` in theory, see #53295.

So this should probably go back to `editor`, maybe in a new `editor/editor_translation.h` file?